docs/design/scheduler-simulate-design.md

```diff
@@ -0,0 +1,32 @@
+# Volcano scheduler simulate
+## background
+* Consider such situation: users changed the parameter of nodeorder plugin and need to know the effect to the production enviroment. For example, after change the mostrequested.weight, if the average wait time of big task is shorter than before, etc.
+* Currently, users must use real cluster to verify the effect, which is very time consuming and need a lot of node resources.
+* Scheduler simulate is an effect solution of the problem above. It runs fast and needs much lesser resources than real cluster.
+## design
+* The simulator includes simulator command, simulate-data-input, simulate-result-output, time simulator, kwok, scheduler
+![scheduler-simulate](images/scheduler-simulate.png) 
+### simulator command
+* scheduler-simulate can be runned by command: vc-scheduler-simulate --running_time=24h --config_path=./config --output_path=./result
+### simulate-data-input
+* Simulate data includes nodes.csv, pods.csv, queues.csv. These files can be put in config_path directory.
+* nodes.csv includes cpu_allocatable, memory_allocatable, label, maxPodNum, etc...
+* pods.csv includes cpu_request, memory_request, runsec, cron, createtime, nodeSelector, priority, queueName, etc...
+* queues.csv includes queueName, quota, etc...
+### simulate-result-output
+#### file
+* output file includes nodes_detail.csv, queues_detail.csv, pods_detail.csv, pods_detail.csv. These files will be put in output_path directory.
+* nodes_detail.csv includes ts, nodeName, cpuRequest, memoryRequest
+* queues_detail.csv includes ts, queueName, cpuRequest, memoryRequest
+* pods_detail.csv includes podName, createTs, scheduleTs, finishTs
+#### metrics
+* output metrics includes volcano_simulator_scheduler_wait_time, volcano_simulator_pod_run_time, volcano_simulator_node_cpu_request, volcano_simulator_node_memory_request which will help user to monitor the schduler simulate by prometheus.
+### time simulator
```

Comment:
I think there is no need to create a time simulator? kwok can do everything, https://kwok.sigs.k8s.io/docs/user/stages-configuration/, you can simulate whatever stage the pod is. Maybe we use kwok+shell script is a better way.